test/IRGen/temporary_allocation/size_too_large.swift

```diff
@@ -3,7 +3,7 @@
 @_silgen_name("blackHole")
 func blackHole(_ value: UnsafeMutableRawPointer?) -> Void
 
-withUnsafeTemporaryAllocation(byteCount: 1, alignment: -1) { buffer in
```

Comment:
Did you mean to delete the original test file here?

Reply:
Yes, it was a duplicate of another file. :)